mobile/src/components/common/icon/common/maps/icon-name-to-icon.map.ts

```diff
@@ -26,5 +35,14 @@ const iconNameToIcon: Record<IconName, FC<SvgProps>> = {
   book: Book,
   uam: UAM,
   trash: Trash,
+  c: C,
+  css: CSS,
+  html: HTML,
+  java: Java,
+  javascript: JavaScript,
+  nodejs: NodeJS,
+  python: Python,
+  react: React,
+  typescript: Typescript,
```

Comment:
Create a local map for this images - category-list/common/maps/category-key-to-image.map.ts, and we don't need type for name, so set just Record<string, FC<SvgProps>>

Reply:
So what type of images should I use? SVG or PNG? Because if svg, I can't use Image, in case that getImageUri helper requires ImageSourcePropType, instead of SvgProps, when i use svg images?

Reply:
Let's use svg

Reply:
So I need to make a new helper to get svg uri, right?

Reply:
But create a separate component for this, don't use common icon